adodbapi/is64bit.py

```diff
@@ -3,15 +3,10 @@
 
 
 def Python():
-    if sys.platform == "cli":  # IronPython
-        import System
-
-        return System.IntPtr.Size == 8
-    else:
-        try:
-            return sys.maxsize > 2147483647
-        except AttributeError:
-            return sys.maxint > 2147483647
+    try:
+        return sys.maxsize > 2147483647
+    except AttributeError:
+        return sys.maxint > 2147483647

```

Comment:
Simplify to "return sys.maxsize > 2147483647"
The AttributeError is only triggered in Python 2.